src/@types/Erc721.ts

```diff
@@ -9,5 +9,5 @@ export interface MetadataAndTokenURI {
   metaDataHash: string
   tokenId: number
   tokenURI: string
-  metadataProofs: MetadataProof[]
+  metadataProofs?: MetadataProof[]
```

Comment:
unfortunately adding only this change is not enough, we need some checks in the hole codebase in every place the interface is used and check for `metadataProofs` existence and if not set we need to set an empty string array `= []` to `metadataAndTokenURI.metadataProofs` , otherwise the contract call will fail if metadataProofs is undefined or null. 
So we need a check similar with this https://github.com/oceanprotocol/ocean.js/blob/v4main/src/tokens/NFT.ts#L1065  in all places the `MetadataAndTokenURI` is used